data/archive/nwerc/1992.yaml

```diff
@@ -1,5 +1,5 @@
 contest:
-  name: NWERC 1992
+  name: WERC 1992
```

Comment:
Same as for 1989. But given the differing style with the years in between (1990 and 1991), perhaps there would have been both an ERC and an NWERC? :thinking: 

![image](https://user-images.githubusercontent.com/9739541/221365925-0d838c0f-7cf5-472e-ac10-b4d2bfdca530.png)

Reply:
Same story as for ERC 89
![IMG20230226080513](https://user-images.githubusercontent.com/5221158/221397141-fba753cb-2019-467d-bc19-f4a0869eee86.jpg)

Reply:
Note that the contest was not ICPC yet, but the ACM Scholastic Programming Contest.